test/parallel/test-output-assert.mjs

```diff
@@ -0,0 +1,33 @@
+import '../common/index.mjs';
+import * as fixtures from '../common/fixtures.mjs';
+import * as snapshot from '../common/assertSnapshot.js';
+import { describe, it } from 'node:test';
+
+function replaceStackTrace(str) {
+  return snapshot.replaceStackTrace(str, '$1at *$7\n');
+}
+
+describe('assertion output', { concurrency: true }, () => {
+  function normalize(str) {
+    return str
+      .replaceAll(snapshot.replaceWindowsPaths(process.cwd()), '')
+      .replaceAll('/', '*')
+      .replaceAll(process.version, '*')
+      .replaceAll(/:\d+/g, '*');
+  }
+
+  const common = snapshot
+    .transform(snapshot.replaceWindowsLineEndings, snapshot.replaceWindowsPaths);
+  const defaultTransform = snapshot.transform(common, replaceStackTrace, normalize);
+
+  const tests = [
+    { name: 'assert/assert_throws_stack.js' },
+    { name: 'assert/internal_assert.js' },
+  ];
```

Comment:
I would prefer if these would be out of the `describe` scope, to avoid the additional nesting, but I don't feel strongly about it.